lib/Doctrine/Common/Cache/FileCache.php

```diff
@@ -164,7 +164,7 @@ protected function doDelete($id)
     {
         $filename = $this->getFilename($id);
 
-        return @unlink($filename) || ! file_exists($filename);
```

Comment:
That won't work though, because it good get deleted in-between that cal and the next one.